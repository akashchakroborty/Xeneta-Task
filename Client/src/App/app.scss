@import '~antd/dist/antd.css';
@import './app.variables.scss';

body {
  margin: 0px;
  box-sizing: border-box;
}

.icon {
  font-size: $primary-font-size;
  color: $primary-color;
}
