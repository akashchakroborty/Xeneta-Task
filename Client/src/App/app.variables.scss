$primary-font-size: 14px;
$primary-color: #00539f;
$white-color: #ffffff;