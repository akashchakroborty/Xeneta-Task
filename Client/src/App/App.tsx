import React from 'react';
import './app.scss';

const App: React.FC = () => {
  return <div>App is loading...</div>;
};

export default App;
