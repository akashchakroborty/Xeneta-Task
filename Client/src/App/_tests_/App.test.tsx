import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('should test the App Component', () => {
  test('should render the component', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
});
