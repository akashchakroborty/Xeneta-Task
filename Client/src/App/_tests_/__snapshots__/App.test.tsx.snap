// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div>
  <Header />
  <Navigation />
  <Divider
    className="divider"
  />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      key="/"
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/quality-check"
      path="/quality-check"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/event-authorisation"
      path="/event-authorisation"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/product-details"
      path="/product-details"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/range-authorisation"
      path="/range-authorisation"
    />
    <Route
      component={[Function]}
      exact={true}
      key="/quality-check-details"
      path="/quality-check-details"
    />
  </Switch>
</div>
`;
